components/gateway/src/main/kotlin/net/corda/p2p/gateway/messaging/SslConfiguration.kt

```diff
@@ -11,11 +11,17 @@ data class SslConfiguration(
      * Property determining how the revocation check will be made for the server certificate
      */
     val revocationCheck: RevocationConfig,
+
+    /**
+     * How to validate TLS certificates in the gateway.
```

Comment:
```suggestion
     * What type of TLS connections to establish with gateways in other clusters.
```
in the same spirit as the comment in `corda-api`